modules/javafx.graphics/src/main/java/com/sun/javafx/font/PrismFontFactory.java

```diff
@@ -209,7 +209,7 @@ private static synchronized PrismFontFactory getFontFactory(String factoryClass)
           createFontFile(String name, String filename,
                          int fIndex, boolean register,
                          boolean embedded,
-                         boolean copy, boolean tracked)
+                         boolean copy)
```

Comment:
might want to change the copyright year to 2025

Reply:
Oh I did miss that file didn't I... I think it's time to make a pre-commit hook or something to take care of that for me